readme.md

```diff
@@ -41,6 +41,6 @@ Sort in ascending or descending order by:
 -   Meta Value
 -   Meta Value Num
 -   Random
--   Menu Order ( props to @jvanja )
+-   Menu Order -- for sorting plugins such as [Post Type Order](https://wordpress.org/plugins/post-types-order/). ( props to @jvanja )
```

Comment:
I didn't realize this was a link to a plugin during my first look.  I thought it was a link to the docs.

Menu order can be enabled for [post types programmatically](https://developer.wordpress.org/reference/functions/register_post_type/#hierarchical) so it's not a strict requirement to have a plugin for this. 

I think I'd prefer not to add the link.